test/middleware/spec.back-link.js

```diff
@@ -81,6 +81,8 @@ describe('Back Links', function () {
     });
 
     it('adds the previous step to res.locals.backLink', function () {
+        req.get.withArgs('referrer').returns('http://example.com/referrer');
+        req.baseUrl = '/referrer';
```

Comment:
> This highlights that backlinks were originally written based on having a baseUrl and feels wrong now.

I would agree, but this pr is intended to fix the immediate problem, not rewrite the backLink middleware